frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/grammar/MacheteParserDefinition.java

```diff
@@ -0,0 +1,65 @@
+package com.virtuslab.gitmachete.frontend.file.grammar;
+
+import com.intellij.lang.ASTNode;
+import com.intellij.lang.ParserDefinition;
+import com.intellij.lang.PsiParser;
+import com.intellij.lexer.Lexer;
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.FileViewProvider;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.TokenType;
+import com.intellij.psi.tree.IFileElementType;
+import com.intellij.psi.tree.TokenSet;
+
+public class MacheteParserDefinition implements ParserDefinition {
+  public static final TokenSet WHITE_SPACES = TokenSet.create(TokenType.WHITE_SPACE);
+  public static final TokenSet COMMENTS = TokenSet.create(MacheteTypes.COMMENT);
+
+  public static final IFileElementType FILE = new IFileElementType(MacheteLanguage.INSTANCE);
+
+  @Override
+  public Lexer createLexer(Project project) {
+    return new MacheteLexerAdapter();
+  }
+
+  @Override
+  public TokenSet getWhitespaceTokens() {
+    return WHITE_SPACES;
+  }
+
+  @Override
+  public TokenSet getCommentTokens() {
```

Comment:
Even if we doesn't support comments this element is part of `ParserDefinition` interface, so we need to provide something here (inb4 it is `@NonNull` ;) )

Reply:
Maybe add this comment to the source code as well